### PR TITLE
Enhance the cloud credentials documentation

### PR DESCRIPTION
**Pull request by irespaldiza**

### CODE REVIEW

File: static/img/amazon-ecr-oidc-config.png
Comment (irespaldiza) [inline static/img/amazon-ecr-oidc-config.png:1]: I wouldn’t show real data from any cluster

File: src/content/admin/cloud-credentials/index.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ sidebar_label: Cloud Credentials
 import Image from "@theme/Image";
 
 In the Admin Dashboard, you can configure Cloud Credentials for accessing your cloud account.
-Cloud Credentials are automatically used by Okteto in your deploy, destroy, and test commands, eliminating the need for developers to manage static credentials.
+Cloud Credentials are automatically used by Okteto in your deploy, destroy, and test commands, eliminating the need of manage static credentials.
```
Comment (irespaldiza) [inline src/content/admin/cloud-credentials/index.mdx:10]: I’ve removed the reference to the developer. Permission management typically falls under the responsibility of the security or platform teams, and I believe it’s more appropriate to direct the message towards Cindy and/or Cindy's team taking care of that task.

File: src/content/admin/cloud-credentials/aws.mdx [hunk 2]
```diff
@@ -37,6 +28,15 @@ Okteto displays the OIDC endpoint of your cluster in the Admin -> General view o
      />
    </p>
 
+```bash
```
Comment (irespaldiza) [inline src/content/admin/cloud-credentials/aws.mdx:31]: I would present it this way: first showing where to obtain the data and then how to use it

File: src/content/admin/cloud-credentials/gcp.mdx [hunk 3]
```diff
@@ -20,10 +20,20 @@ GCP credentials are configured using [Workload Identity Federation](https://clou
 
 As a first step let's create a Workload Identity Pool. A Workload Identity Pool is a group of identities that can be used to access GCP resources.
 
+Okteto displays the OIDC endpoint of your cluster in the Admin -> General view of the Dashboard:
```
Comment (irespaldiza) [inline src/content/admin/cloud-credentials/gcp.mdx:23]: I would present it this way: first showing where to obtain the data and then how to use it.

File: src/content/admin/cloud-credentials/gcp.mdx [hunk 4]
```diff
@@ -20,10 +20,20 @@ GCP credentials are configured using [Workload Identity Federation](https://clou
...
+       src={require("@site/static/img/amazon-ecr-oidc-config.png").default}
+       alt="OIDC configuration in admin general view"
+     />
+   </p>
+
 ```bash
 PROJECT_ID=my-project-12345
 POOL_ID=okteto-pool
 AUDIENCE=deploy.okteto.dev
+OIDC_ENDPOINT=https://container.googleapis.com/v1/projects/my-project-12345/locations/us-central1/clusters/development
```
Comment (irespaldiza) [inline src/content/admin/cloud-credentials/gcp.mdx:36]: I believe it’s better to use a variable to make it clear that this is a value that needs to be configured.

File: src/content/admin/cloud-credentials/gcp.mdx [hunk 5]
```diff
@@ -38,29 +48,14 @@ Within the pool, let's register the kubernetes cluster as an identity provider.
...
-       src={require("@site/static/img/amazon-ecr-oidc-config.png").default}
-       alt="OIDC configuration in admin general view"
-     />
-   </p>
-
 Traditionally, `AUDIENCE` is the client id of the requester. You'll only exchange token for these audiences. It is the `aud` field of the JWT payload.
 
 We recommend creating a different/dedicated audience for each Okteto instance and region used, even if you use the same identity provider:
 
-```
```
Comment (irespaldiza) [inline src/content/admin/cloud-credentials/gcp.mdx:58]: It seems very odd to show AWS nomenclature when both the displayed cluster and the cloud provider to be federated are in GCP.

**Review by codyjlandstrom — APPROVED** [review]

**Review by pchico83 — APPROVED** [review]

**Pull request merged**